python/tests/test_pipeline_builder.py

```diff
@@ -647,6 +649,175 @@ def test_input_json1(self):
 
         pipeline.delete()
 
+    def test_issue2142(self):
+        sql = f"""
+        CREATE TABLE t0 (c1 INT);
+        CREATE VIEW v0 AS SELECT * FROM t0;
+        """
+
+        pipeline = PipelineBuilder(TEST_CLIENT, name="test_issue2142", sql=sql).create_or_replace()
+
+        data = [{"c1": None}, {"c1": 1}]
+
+        out = pipeline.listen("v0")
+
+        pipeline.start()
+        pipeline.input_json("t0", data=data)
+        pipeline.wait_for_completion(True)
+
+        out_data = out.to_dict()
+
+        for datum in data:
+            datum.update({"insert_delete": 1})
+
+        assert out_data == data
+
+        pipeline.delete()
+
+    def test_pandas_binary(self):
```

Comment:
do these need some annotations to run as tests?

Reply:
No, no annotations necessary.